app/services/time_slot_computer.rb

```diff
@@ -29,7 +29,11 @@ def run
   private
 
   def format_slot(start, finish)
-    "#{start.iso8601}/#{finish.iso8601}"
+    # this becomes a dictionary key in a serialized thing that
+    # goes in the db.  The `::` seems to be the way to get all the
+    # serialize/deserialize between here and the front end to work
+    # out nicely
+    "#{start.to_i}::#{finish.to_i}"
```

Comment:
moved from iso to ints because the iso format as a key in a serialized dictionary in the db ran through all kinds of weird encoding making the keys change their shape as they went in and out of different parts of the app.  Seconds since the Epoch is equally accurate as a time and maintains timezone info correctly (cuz it's just time)